Pollo/Views/Cards/MCResultCell.swift

```diff
@@ -169,48 +169,44 @@ class MCResultCell: UICollectionViewCell {
-            containerView.layer.cornerRadius = highlightViewCornerRadius
-            optionLabel.textColor = .mediumGrey
             numSelectedLabel.isHidden = true
             dotView.isHidden = false
-            let isSelected = resultModel.isSelected
-            selectedDotView.isHidden = !isSelected
-            selectedImageView.isHidden = selectedDotView.isHidden
-            if let correctAnswer = correctAnswer, !correctAnswer.isEmpty {
-                if answer == correctAnswer {
-                    if isSelected {
-                        selectedDotView.backgroundColor = .clear
-                        selectedImageView.image = UIImage(named: correctImageName)
-                    }
-                    showCorrectAnswer = true
-                    highlightView.backgroundColor = .lightGreen
-                    highlightView.layer.borderColor = UIColor.polloGreen.cgColor
-                } else {
-                    if isSelected {
-                        selectedDotView.backgroundColor = .clear
-                        selectedImageView.image = UIImage(named: incorrectImageName)
-                    }
-                    highlightView.backgroundColor = .lightGrey
-                    highlightView.layer.borderColor = UIColor.coolGrey.cgColor
+            optionLabel.textColor = .mediumGrey
+        }
+        containerView.backgroundColor = .clear
+        containerView.layer.borderWidth = highlightViewBorderWidth
+        containerView.layer.cornerRadius = highlightViewCornerRadius
+        let isSelected = resultModel.isSelected
+        selectedDotView.isHidden = !isSelected
+        selectedImageView.isHidden = selectedDotView.isHidden
+        if let correctAnswer = correctAnswer, !correctAnswer.isEmpty {
+            if answer == correctAnswer {
+                if isSelected {
+                    selectedDotView.backgroundColor = .clear
+                    selectedImageView.image = UIImage(named: correctImageName)
                 }
+                showCorrectAnswer = true
+                highlightView.backgroundColor = .lightGreen
+                highlightView.layer.borderColor = UIColor.polloGreen.cgColor
             } else {
                 if isSelected {
-                    selectedDotView.backgroundColor = .coolGrey
-                    selectedImageView.image = nil
+                    selectedDotView.backgroundColor = .clear
+                    selectedImageView.image = UIImage(named: incorrectImageName)
                 }
                 highlightView.backgroundColor = .lightGrey
                 highlightView.layer.borderColor = UIColor.coolGrey.cgColor
             }
+        } else {
+            if isSelected {
+                selectedDotView.backgroundColor = .coolGrey
+                selectedImageView.image = nil
+            }
+            highlightView.backgroundColor = .lightGrey
+            highlightView.layer.borderColor = UIColor.coolGrey.cgColor
```

Comment:
Is there some way to reduce the nested if/else?
No idea if this works, but something like this:
```
highlightView.backgroundColor = .lightGrey
highlightView.layer.borderColor = UIColor.coolGrey.cgColor
selectedDotView.backgroundColor = isSelected ? .coolGrey : .clear
selectedImageView.image = nil
if let correctAnswer = correctAnswer, !correctAnswer.isEmpty {
              let selectedCorrectAnswer = answer == correctAnswer
              if isSelected {
                      selectedDotView.backgroundColor = .clear
                      selectedImageView.image = UIImage(named: selectedCorrectAnswer  ? correctImageName : incorrectImageName)
              }
              if selectedCorrectAnswer {
                  showCorrectAnswer = true
                  highlightView.backgroundColor = .lightGreen
                  highlightView.layer.borderColor = UIColor.polloGreen.cgColor
              }
          }
```

Reply:
lol I didn't even realize how redundant the code was before